certbot-apache/tests/autohsts_test.py

```diff
@@ -2,13 +2,13 @@
 """Test for certbot_apache._internal.configurator AutoHSTS functionality"""
 import re
 import unittest
-import mock
+
 # six is used in mock.patch()
```

Comment:
this comment is about `six` and should be moved down with it

Reply:
Done.